content/api/commands/exec.md

```diff
@@ -210,7 +210,7 @@ exist.</li></List>
 
 ## Command Log
 
-**_List the contents of the default `cypress.json` configuration file_**
+**_List the contents of the default configuration file_**
```

Comment:
The new default configuration file can be either `cypress.config.js` or `cypress.config.ts` so it's probably not a good idea to refer to the "default configuration file" anywhere. Also, on lines 217 and 219 of this file, `cypress.json` is referenced, which will need to change. We will need to refer to a different file (and different language) for any examples that mention `cypress.json`.

Reply:
@amirrustam @jaffrepaul any thoughts on what the generic file we reference in examples like this should be? It would be great if it was a file we knew was always going to be there... maybe `package.json` or `README.md`?

Reply:
Eg. "List the contents of your `package.json` file" and then change `cypress.json` -> `package.json` in the example

Reply:
> it's probably not a good idea to refer to the "default configuration file" anywhere.

I agree, but if the `--config-file` CLI flag is sticking around then there would be a "default" configuration file. Thoughts?

Reply:
> Eg. "List the contents of your package.json file" and then change cypress.json -> package.json in the example

This would work.

Reply:
Fixed in https://github.com/cypress-io/cypress-documentation/pull/4093/commits/ed5eb544e51a284ebfc093b2d9fee106e08cd032